static/src/javascripts/bootstraps/enhanced/main.js

```diff
@@ -58,6 +59,8 @@ const bootEnhanced = (): void => {
         ],
 
         ['enrich-acquisition-links', initAcquisitionsLinkEnrichment],
+
+        ['remote-epics', fetchAndRenderEpic ]
```

Comment:
make the request to support-dotcom-components here, rather than as part of frontend's A/B testing mechanism